source/blog/tags.html

```diff
@@ -8,6 +8,17 @@
 
 <div>
 {% for tag,posts in data.posts_tags %}
-<a href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
+
+<!-- conditions to hide author tag names on the tags page --> 
+  {% if tag == "nikki-stevens" %} 
+    <a class="hide" href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a> 
+  {% elseif tag == "mike-klass" %}
+     <a class="hide" href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a> 
+  {% elseif tag == "jason-savino" %}
+     <a class="hide" href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
+ 
+  {% else %} 
+  <a href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
```

Comment:
ALSO, I'm just now testing this and it doesn't seem to work. :( https://dl.dropboxusercontent.com/s/l7h0p77g6c7ymsm/2015-08-18%20at%209.06%20AM%202x.png?dl=0